its/ruling/src/test/resources/jdk6/squid-SwitchLastCaseIsDefaultCheck.json

```diff
@@ -162,27 +159,6 @@
 'jdk6:java/io/StreamTokenizer.java':[
 644,
 ],
-'jdk6:java/lang/CharacterData00.java':[
```

Comment:
The check now requires semantic, so all the issues within `java.lang` package are lost.